scripts/pin_the_slow_way.py

```diff
@@ -41,16 +41,16 @@
           'boost': 'boost 1.65.1',  # 1.61.0
           'boost-cpp': 'boost-cpp 1.65.1',  # NA
           'bzip2': 'bzip2 1.0.*',  # 1.0.6
-          'cairo': 'cairo 1.14.*',  # 1.12.18
+          'cairo': 'cairo 1.14.*',  # 1.14.10
           'curl': 'curl >=7.44.0,<8',  # 7.54.1
           'ffmpeg': 'ffmpeg >=3.2.3,<3.2.6',  # NA
           'flann': 'flann 1.9.1',  # NA
-          'fontconfig': 'fontconfig 2.12.*',  # 2.12.1
-          'freetype': 'freetype 2.7',  # 2.5.5
+          'fontconfig': 'fontconfig 2.12.*',  # 2.12.4
+          'freetype': 'freetype 2.8.1',  # 2.8
           'gdal': 'gdal 2.2.*',  # 2.2.0
           'geos': 'geos 3.6.2',  # 3.5.0
           'giflib': 'giflib 5.1.*',  # NA
-          'glib': 'glib 2.51.*',  # 2.50.2
+          'glib': 'glib 2.55.*',  # 2.53.6
```

Comment:
Sending this version in a moment. See https://abi-laboratory.pro/tracker/timeline/glib/

Reply:
There seem to be two removed symbols between 2.51 and 2.55. It is probably worth updating the pin.